Spark/traffic_speed_bands.py

```diff
@@ -0,0 +1,19 @@
+from pyspark.sql import SparkSession
+from pyspark.sql import functions as F
+
+def main():
+    spark = SparkSession.builder.getOrCreate()
+    dataframe = spark.read.format("json").load("s3://cs4225-traffic-speed-bands").withColumn("timestamp_raw", F.input_file_name())
+    data = []
+    array_raw = dataframe.select("value").collect()
+    array_timestamp_raw = dataframe.select("timestamp_raw").collect()
```

Comment:
Not sure but it seems like for this part it loads the entire data into ram, which is using python API instead of spark API, hence the processing is not distributed. Same for other files.

Reply:
> Can also elaborate on why Traffic_speed_bands is unable to work?
data seems to be too huge to process, changing executor and driver memory was able to allow carpark data to be able to be processed so not sure what's the exact problem.